replication-api/src/main/java/org/codice/ditto/replication/api/QueryService.java

```diff
@@ -0,0 +1,9 @@
+package org.codice.ditto.replication.api;
+
+import java.util.List;
+import org.codice.ditto.replication.api.data.Query;
+
+public interface QueryService {
```

Comment:
All classes and public methods / attributes should be documented properly. Sometimes that can be very simple in deed.

Reply:
Everything I've read agrees with you but nothing explains the value in adding javadoc to a simple method like a getter. It seems like it's just more text for the developer to skip over. So my question is, why should they have javadoc? I could be missing something because I very rarely look at javadoc.